overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -433,3 +434,23 @@ func (s *servicectlSuite) TestQueuedCommandsSingleLane(c *C) {
 	c.Check(laneTasks[14].Summary(), Equals, "start of [test-snap.test-service]")
 	c.Check(laneTasks[15].Summary(), Equals, "restart of [test-snap.test-service]")
 }
+
+func (s *servicectlSuite) TestServices(c *C) {
+	restore := systemd.MockSystemctl(func(args ...string) (buf []byte, err error) {
+		c.Assert(args[0], Equals, "show")
+		return []byte(`Id=snap.test-snap.test-service.service
+Type=simple
+ActiveState=active
+UnitFileState=enabled
+`), nil
+	})
+	defer restore()
+
+	stdout, stderr, err := ctlcmd.Run(s.mockContext, []string{"services"}, 0)
```

Comment:
Could you also add a test for a case where snap has two services and we query just one of them?

Reply:
done (and now sorting the output, thank you!)